jenkins/run-pytests

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+prog="$(basename "${0}")"
+progdir="$(realpath -e $(dirname "${0}")/..)"
+
+source $(dirname "${0}")/python-setup.sh
+
+printf -- "black --check ...\n"
+black --check .
+black_sts=${?}
+if [[ ${black_sts} -ne 0 ]]; then
+    printf -- "FAILED: black --check .\n" >&2
+fi
+printf -- "\nflake8 --verbose ...\n"
+flake8 --verbose .
+flake_sts=${?}
+if [[ ${flake_sts} -ne 0 ]]; then
+    printf -- "FAILED: flake8 --verbose .\n" >&2
+fi
```

Comment:
Black and Flake8 are already being run by [`run_unittests`](https://github.com/riya-17/pbench/blob/09630364d165f4fc52b2e6d1d56fcd59c49715f1/jenkins/run-unittests#L10) on `b0.69`.  If you would rather run them from here, I'm fine with that, but you should remove them from there.

Reply:
if I look according to `main` branch they are in run-pytests, so, I will add them there and remove from `run-unittests`